bin/select_targets

```diff
@@ -20,7 +21,18 @@ ap.add_argument("src", help="Tractor/sweeps file or root directory with tractor/
 ap.add_argument("dest", help="Output target selection file")
 ap.add_argument('-v', "--verbose", action='store_true')
 ap.add_argument('-c', "--check", action='store_true',help="Process tractor/sweeps files to check for corruption, without running full target selection")
-ap.add_argument("--sandbox", action='store_true',help="Apply the sandbox target selection algorithms.")
+ap.add_argument('-s', "--starmask", action='store_true',help="Mask the targets for bright objects")
+ap.add_argument("--starmaskfile", 
+                help='File containing bright object mask if -s (--starmask) is sent. If not sent, star mask is built from files in src directory')
+ap.add_argument("--starmaskbands", 
+                help='Bands to use to build the star mask if -s (--starmask) is sent but --starmaskfile is not (e.g. GRZ)',
+                default="GRZ")
+ap.add_argument("--starmaskmaglim",
+                help='Magnitude limits for building the star mask if -s (--starmask) is sent but --starmaskfile is not (e.g. 10,10,10)',
+                default="10,10,10")
+ap.add_argument("--starmaskoutfile",
+                help='Output file name to which to write star mask if -s (--starmask) is sent but --starmaskfile is not')
+ap.add_argument("--sandbox", action='store_true',help="Apply the sandbox target selection algorithms")
```

Comment:
This set of options seems to overload what `select_targets` is supposed to do, conflating selecting targets vs building bright star masks.  I'd prefer a separate executable focused on building the star mask and then have that mask file used as an input to select_targets.  Building the mask might grow in complexity and number of options and may eventually not even use the same inputs as `select_targets`.